src/borg/helpers.py

```diff
@@ -1527,12 +1531,21 @@ def get_item_data(self, item):
             item_data[key] = self.call_keys[key](item)
         return item_data
 
-    def calculate_num_chunks(self, item):
-        return len(item.get('chunks', []))
+    def sum_unique_chunks_property(self, property_func, item):
+        """
+        sum unique chunks properties, a unique chunk is a chunk which is referenced globally as often as it is in the
+        item
 
-    def calculate_unique_chunks(self, item):
+        item: The item to sum it's unique chunks
```

Comment:
"its"

as this piece of grammar seems to be not too well known:

it is very easy to differentiate between "its" and "it's".

"it's" is a contraction for "it is", so just try if "it is" fits in the sentence.
if not, you wanted "its" (which is a possessive pronoun, like "his", "her").

Related:

http://www.apostrophecatastrophes.com/

http://www.angryflower.com/247.html
(here we see where the confusion comes from: possessive form for nouns is with apostrophe, but the possessive pronoun "its" is without ...)